hw/hw_08_task_02.py

```diff
@@ -0,0 +1,34 @@
+import sqlite3
+from typing import Tuple
+
+
+# flake8: noqa: S608
+class TableData:
+    def __init__(self, database_name: str, table_name: str):
+        self.database_name = database_name
+        self.table_name = table_name
+
+    @staticmethod
+    def connect_database(database_name: str) -> sqlite3.Cursor:
+        conn = sqlite3.connect(database_name)
```

Comment:
Which method is always called in examples and never in this code?

Reply:
I'm not sure, but maybe you are talking about closing connection?

Reply:
Exactly!